### PR TITLE
Add Fleet Server install steps on K8s and Docker

### PR DESCRIPTION
**Pull request by kilfoyle**
This updates the docs with the extra steps required to install Elastic Agent as a Fleet Server in containerized environments.

**On Kubernetes**
 - On the page "Run Elastic Agent on Kubernetes managed by Fleet" this updates "[Step 3: Enroll Elastic Agent to the policy](https://www.elastic.co/guide/en/fleet/master/running-on-kubernetes-managed-by-fleet.html#_step_3_enroll_elastic_agent_to_the_policy)" to add the following:
    Screenshot 2024-09-18 at 12 38 25 PM

**In Docker**
 - On the page "Run Elastic Agent in a container" this updates "[Step 4: Run the Elastic Agent image](https://www.elastic.co/guide/en/fleet/master/elastic-agent-container.html#_step_4_run_the_elastic_agent_image)"  to add the following:
    Screenshot 2024-09-18 at 12 40 19 PM

---

 Rel: https://github.com/elastic/ingest-docs/issues/1075

### CODE REVIEW

**Review by michel-laterman — APPROVED** [review]

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc [hunk 1]
```diff
@@ -67,8 +67,22 @@ To specify different destination/credentials, change the following parameters in
...
+[source,yaml]
+------------------------------------------------
+- name: FLEET_SERVER_ENABLE
+  value: "true" <1>
+- name: FLEET_SERVER_ELASTICSEARCH_HOST
+  value: "<elasticsearch-host>" <2>
+- name: FLEET_SERVER_SERVICE_TOKEN
+  value: "<service-token>" <3>
+------------------------------------------------
+<1> Set to `1` to bootstrap {fleet-server} on this {agent}. This automatically forces {fleet} enrollment as well.
```
Comment (kilfoyle) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc:81]:
```suggestion
<1> Set to `true` to bootstrap {fleet-server} on this {agent}. This automatically forces {fleet} enrollment as well.
```
I just noticed that the description doesn't match the example. @michel-laterman Should `FLEET_SERVER_ENABLE` be set to `true` or to `1`, or are both okay?

Comment (michel-laterman) [reply docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc:81]: Both values are accepted and will be parsed as a bool `true` value, so I guess we should be consistent with whatever we have been using in the docs

File: docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc [hunk 2]
```diff
@@ -95,6 +95,18 @@ docker run --rm docker.elastic.co/beats/elastic-agent:{version} elastic-agent co
...
 
+If you need to run {fleet-server} as well, adjust the `docker run` command above by adding these environment variables:
+
+[source,yaml]
+----
+  --env FLEET_SERVER_ENABLE=true \ <1>
+  --env FLEET_SERVER_ELASTICSEARCH_HOST=<elasticsearch-host> \ <2>
+  --env FLEET_SERVER_SERVICE_TOKEN=<service-token> <3>
+----
+<1> Set to `1` to bootstrap {fleet-server} on this {agent}. This automatically forces {fleet} enrollment as well.
```
Comment (kilfoyle) [inline docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc:106]:
```suggestion
<1> Set to `true` to bootstrap {fleet-server} on this {agent}. This automatically forces {fleet} enrollment as well.
```

**Comment by kilfoyle** [comment]
@elasticmachine run docs-build

**Pull request merged**